packages/editor/src/components/post-card-panel/index.js

```diff
@@ -62,7 +74,10 @@ export default function PostCardPanel( { className, actions } ) {
 			__( 'Last edited %s.' ),
 			humanTimeDiff( modified )
 		);
-
+	const showPostContentInfo =
+		! [ TEMPLATE_POST_TYPE, TEMPLATE_PART_POST_TYPE ].includes(
+			postType
+		) && ! isPostsPage;
```

Comment:
How wrong of me is it to want to avoid instantiating an array unless necessary? :P 

```suggestion
		! isPostsPage &&
		! [ TEMPLATE_POST_TYPE, TEMPLATE_PART_POST_TYPE ].includes( postType );
```